Clappr.xcodeproj/project.pbxproj

```diff
@@ -402,7 +409,7 @@
 		96D362B21D41339400CCB866 /* Playback */ = {
 			isa = PBXGroup;
 			children = (
-				96D362B31D41339400CCB866 /* AVFoundationPlayback.swift */,
+				9E18A7401FF5512E00C706F4 /* AVFoundation */,
```

Comment:
@CristianDRM not sure if AVFoundation should be used in a Clappr's folder name as it might cause confusion with Apple's framework name. What do you think?